@stellar/anchor-tests/src/tests/sep31/transactions.ts

```diff
@@ -97,6 +112,97 @@ const canCreateTransaction: Test = {
       // but to satisfy TypeScript we make these checks here.
       throw "improperly configured";
     }
+
+    async function pollCustomerAccepted(
+      getCustomerNetworkCall: NetworkCall,
+      timeout: number,
+    ) {
+      for (let i = 0; i < timeout; i++) {
+        const resBody = await makeRequest(
+          getCustomerNetworkCall,
+          200,
+          result,
+          "application/json",
+        );
+        if (resBody.status === "ACCEPTED") {
+          return;
+        }
+        await new Promise((resolve) => setTimeout(resolve, 1000));
+      }
+      throw "timed out pulling customer for ACCEPTED status";
+    }
+
+    function generateGetCustomerNetworkCall(
+      kycServerUrl: string,
+      customerId: string,
+      token: string,
+    ): NetworkCall {
+      const requestParamsObj: Record<string, string> = {
+        id: customerId,
+      };
+
+      const getCreateCustomerType = (config: Config): string | undefined =>
+        config?.sepConfig?.["12"]?.customers?.[
+          config?.sepConfig?.["12"]?.createCustomer
+        ].type;
+      const customerType = getCreateCustomerType(config);
```

Comment:
Why create a function that is only used once immediately after its definition?

```suggestion
      const customerType = config?.sepConfig?.["12"]?.customers?.[
        config?.sepConfig?.["12"]?.createCustomer
      ].type;
```